uiowa/uiowa
# Entity print enabled but hidden by default

Fixes #3204 

# How to test
- Sync a site, ensure that there are no issues, check basic site functionality.
  - Test that routes like `/admin/config/content/entityprint` are only available to admins. 
- Sync admissions and test as webmaster:
  - https://admissions.uiowa.ddev.site/kiosk
  - https://admissions.uiowa.ddev.site/areas-of-study/print-links
- Sync a v2 site and check that entity print stuff is hidden on `/admin/structure/paragraphs_type/section/display`.
